### PR TITLE
Added functionality for displaying a screen-space hit effect

### PR DESCRIPTION
**Pull request by ericrius1**
This PR adds the ability to display a screen-space hit effect, and includes an example script of how to toggle it on and off. Will be useful in game experiences where collisions occur, to signal to user they've been hit.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: dcb20120703a48df6a9fa59072db819ea03b2435 available at: https://gist.github.com/2aa24f4dd573582fde07<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.dmg

File: libraries/render-utils/src/HitEffect.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
+//
+//  HitEffect.cpp
+//  interface/src/renderer
+//
+//  Created by Andrzej Kapolka on 7/14/13.
```
Comment (ZappoMan) [inline libraries/render-utils/src/HitEffect.cpp]: update with your name and new date

**Comment by PhilipRosedale** [comment]
@ericrius1 Can you show me this tomorrow, and can we get pulled?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 469bace7ca4ec8d00ce2f6453f2d1d61df613e65 available at: https://gist.github.com/2aa24f4dd573582fde07<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.dmg

**Comment by samcake** [comment]
Eric your branch has conflict
you need to do a merge and resolve

**Comment by samcake** [comment]
SO reviewed, tested on windows successfully
Just need to resolve the merge conflict

THat's it, let me know when you resolve it Eric

**Comment by hifi-gustavo** [comment]
Build console output for commit: 697cb4c9d01cd9bf87cbf6f5be923307f4ea32c3 available at: https://gist.github.com/2aa24f4dd573582fde07<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 65e04337bf7abb58ad15e0f3159d236173a13812 available at: https://gist.github.com/2aa24f4dd573582fde07<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5369.dmg

**Pull request merged**